doc/pdo.rst

```diff
@@ -73,27 +76,38 @@ starts at 1, not 0)::
         for var in message:
             print('%s = %d' % (var.name, var.raw))
 
-    node.pdo.tx[4].add_callback(print_speed)
+    node.tpdo[4].add_callback(print_speed)
     time.sleep(5)
 
     # Stop transmission of RxPDO
-    node.pdo.rx[4].stop()
+    node.rpdo[4].stop()
 
 
 API
 ---
 
-.. autoclass:: canopen.pdo.PdoNode
+.. autoclass:: canopen.pdo.PdoBase
    :members:
 
-   .. py:attribute:: rx
+   .. py:attribute:: map
 
-      The :class:`canopen.pdo.Maps` object representing the receive PDO maps.
+      The :class:`canopen.pdo.Mcps` object representing map associated with the instantiated PDO (transmit or receive).
 
-   .. py:attribute:: tx
 
-      The :class:`canopen.pdo.Maps` object representing the transmit PDO maps.
+.. autoclass:: canopen.pdo.PDO
+   :members:
+
+
+.. autoclass:: canopen.pdo.RPDO
+   :members:
+
+
+.. autoclass:: canopen.pdo.TPDO
+   :members:
+
+   .. method:: stop()
```

Comment:
Won't this show up using autodoc like the other methods?

Reply:
I think so. I have it removed from the pdo.rst